app/controllers/cloud_subnet_controller.rb

```diff
@@ -52,17 +52,38 @@ def cloud_subnet_form_fields
     }
   end
 
+  def cloud_subnet_networks_by_ems
+    assert_privileges("cloud_subnet_new")
+    networks = []
+    available_networks = CloudNetwork.where(:ems_id => params[:id]).find_each
```

Comment:
this `find_each` + `each` combination looks wrong

Reply:
Yeah, not optimal, fixed it, thanks